okhttp/src/test/java/okhttp3/CacheCorruptionTest.kt

```diff
@@ -0,0 +1,141 @@
+package okhttp3
```

Comment:
gotta spotless our copyright thingies

Reply:
MVP would just be a IFTTT script that comments on my PR when I add a new file matching *.kt.  Also asks me to use complete sentences in comments. Asks me not to inline expressions for function declarations.